metadata/ComputerVision__6__metadata.json

```diff
@@ -1,26 +1,26 @@
 {
-  "changeLog": "Release v2023.04.3",
+  "changeLog": "Release v2023.04.4",
   "cpu": 0,
-  "description": "Backend server for the Computer Vision solution that detects User Interface Elements from a provided Application screenshot",
+  "description": "Backend server for the AI Computer Vision solution that detects User Interface Elements from a provided Application screenshot",
   "displayName": "ComputerVision",
   "gpu": 1,
-  "inputDescription": "Please use the Computer Vision Activities included in the UIAutomation package. For more information please visit the official documentation. https://docs.uipath.com/activities/other/latest/user-guide/computer-vision ",
+  "inputDescription": "Please use the Computer Vision Activities included in the UIAutomation package. For more information please visit the official documentation. https://docs.uipath.com/activities/other/latest/ui-automation/computer-vision-activities ",
   "inputType": "JSON",
   "memory": 0,
   "mlPackageLanguage": "PYTHON36_CV",
   "name": "ComputerVision",
-  "outputDescription": "Please use the Computer Vision Activities included in the UIAutomation package. For more information please visit the official documentation. https://docs.uipath.com/activities/other/latest/user-guide/computer-vision ",
+  "outputDescription": "Please use the Computer Vision Activities included in the UIAutomation package. For more information please visit the official documentation. https://docs.uipath.com/activities/other/latest/ui-automation/computer-vision-activities ",
   "processorType": "GPU",
   "projectId": "[project-id]",
   "retrainable": false,
   "stagingUri": "[staging-uri]",
   "projectName": "UiPath Computer Vision",
-  "projectDescription": "Backend server for the Computer Vision solution that detects User Interface Elements from a provided Application screenshot",
+  "projectDescription": "Backend server for the AI Computer Vision solution that detects User Interface Elements from a provided Application screenshot",
   "tenantName": "UiPath",
   "minAIFabricVersion": "22.10",
   "languageVersion": 3,
   "version": 5,
-  "customVersion": "23.04.3",
-  "imagePath": "cv2304:23.4.10.2",
+  "customVersion": "23.04.4",
+  "imagePath": "cv2304:23.4.10.4",
```

Comment:
I'm in the process of building and pushing this one.

Reply:
https://uipath.visualstudio.com/ML%20Platform/_build/results?buildId=4775543&view=results